python/pyarrow/tests/parquet/test_dataset.py

```diff
@@ -352,6 +352,25 @@ def test_filters_cutoff_exclusive_datetime(tempdir, use_legacy_dataset):
     assert result_df['dates'].values == expected
 
 
+@pytest.mark.pandas
+def test_filters_inclusive_datetime(tempdir):
+    # ARROW-11480
+    path = tempdir / 'timestamps.parquet'
+
+    pd.DataFrame({
+        "dates": pd.date_range("2020-01-01", periods=10, freq="D"),
+        "id": range(10)
+    }).to_parquet(path, use_deprecated_int96_timestamps=True)
+
+    import datetime
```

Comment:
Small nit: datetime is already imported at the top, so can be removed here

Reply:
will do